assets/js/common/Tags/Tags.stories.jsx

```diff
@@ -15,3 +15,7 @@ export function Populated(args) {
 export function Empty(args) {
   return <Tags tags={[]} {...args} />;
 }
+
+export function WithValidation(args) {
```

Comment:
As validation is embedded in the tag it would be possible to test it in any other story, so I guess we might not need this.

Besides this this story is the same as the previous `Empty` one :smile:

Reply:
good catch, I removed the story in https://github.com/trento-project/web/pull/2696/commits/3fb66a886c417616600bcac4ffd98f939b2e0bde